src/components/App.js

```diff
@@ -121,16 +122,14 @@ function App() {
             isChecked={hasSymbols}
             onToggle={setHasSymbols}
           />
-          <label>
-            <input
-              type="number"
-              min="4"
-              max="99"
-              value={length}
-              onChange={e => setLength(+e.target.value)}
-            />
-            Length
-          </label>
+
+          <Range
+            id="length"
+            min={4}
```

Comment:
A `label` or an `aria-label` prop is missing.

Reply:
Done